src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,12 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        User[] users = UserService.getUsers();
```

Comment:
`UserService` must be a field of Class
```suggestion
        User user = userService.findByEmail(email);
```